DTW/Bridge DTF/map.json

```diff
@@ -0,0 +1,96 @@
+{
+	"name": "Bridge DTF",
+	"authors": ["Clijmart"],
+	"version": "1.0",
+	"gametype": "DTM",
+	"teams": [
+		{
+			"id": "yellow",
+			"name": "Benny",
+			"color": "yellow",
+			"max": 50,
+			"min": 1
+		},
+		{
+			"id": "gold",
+			"name": "Lucas",
+			"color": "gold",
+			"max": 50,
+			"min": 1
+		}
+	],
+	"spawns": [
+		{ "teams": ["spectators"], "coords": "0.5, 51, 0.5, 0" },
```

Comment:
Remove the space after the { but before the }. Remove the last 0 as well, you don't need to add the yaw if it's 0.